server/store.go

```diff
@@ -206,3 +214,49 @@ func (p *Plugin) removeSuperUserToken() error {
 
 	return nil
 }
+
+func (p *Plugin) storeZoomChannelSettings(channelID string, zoomChannelSettingsMapValue ZoomChannelSettingsMapValue) error {
+	bytes, appErr := p.API.KVGet(zoomChannelSettings)
```

Comment:
I typically like to use `b` as the var name for `[]byte`. There exists a package `bytes` in the Go standard library, so this makes sure we don't shadow that library by accident